packages/protocol/scripts/bash/check-versions.sh

```diff
@@ -59,6 +59,7 @@ fi
 
 echo " - Return to original git ref"
 git checkout $ORIGINAL_GIT_REF > $LOG_FILE
+yarn build:ts
```

Comment:
Is this build neccessary since we are running ts-node?

Reply:
Good question, was thinking myself that it shouldn't be necessary, but I'm pretty sure I remember needing this, at least locally, for the script to run.